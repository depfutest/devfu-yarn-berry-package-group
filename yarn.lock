# This file is generated by running "yarn install" inside your project.
# Manual changes might be lost - proceed with caution!

__metadata:
  version: 5
  cacheKey: 8

"focus-trap-react@npm:^8.1.0":
  version: 8.11.3
  resolution: "focus-trap-react@npm:8.11.3"
  dependencies:
    focus-trap: ^6.9.4
  peerDependencies:
    prop-types: ^15.8.1
    react: ">=16.0.0"
    react-dom: ">=16.0.0"
  checksum: d293b0645b91b2e7c4dee2329ab4aaab50c09d2f935478702f5049f1801db5ac647e6547e96af5e57475b63f3ee7d1fa86026a4539ac98175f5da0d61d8ac89f
  languageName: node
  linkType: hard

"focus-trap@npm:^6.9.4":
  version: 6.9.4
  resolution: "focus-trap@npm:6.9.4"
  dependencies:
    tabbable: ^5.3.3
  checksum: 0b4cebcc11010bd9397731092bd59a981e838c710c6497374ba70571875a14ab27c0db7d60f36005ec01bdabc3b9cfeda11d30ddf5b8874596dcc95e18913b3b
  languageName: node
  linkType: hard

"jquery@npm:^2.1.2":
  version: 2.2.4
  resolution: "jquery@npm:2.2.4"
  checksum: dc8b72b50de9b31699314e36d01fa2ed5f9b6cfa09e51c94ad07f3021e04973707247a051e317283ef96d3f10fff7542947f1916bbcbb2d9ecabff99d1a9cd58
  languageName: node
  linkType: hard

"js-tokens@npm:^3.0.0 || ^4.0.0":
  version: 4.0.0
  resolution: "js-tokens@npm:4.0.0"
  checksum: 8a95213a5a77deb6cbe94d86340e8d9ace2b93bc367790b260101d2f36a2eaf4e4e22d9fa9cf459b38af3a32fb4190e638024cf82ec95ef708680e405ea7cc78
  languageName: node
  linkType: hard

"loose-envify@npm:^1.1.0, loose-envify@npm:^1.3.1":
  version: 1.4.0
  resolution: "loose-envify@npm:1.4.0"
  dependencies:
    js-tokens: ^3.0.0 || ^4.0.0
  bin:
    loose-envify: cli.js
  checksum: 6517e24e0cad87ec9888f500c5b5947032cdfe6ef65e1c1936a0c48a524b81e65542c9c3edc91c97d5bddc806ee2a985dbc79be89215d613b1de5db6d1cfe6f4
  languageName: node
  linkType: hard

"no-scroll@npm:^2.1.1":
  version: 2.1.1
  resolution: "no-scroll@npm:2.1.1"
  checksum: fe57464f42c806ed01af7a43a56dcb9799ebef8fd3fa86d6367769b5b1952b173ef01965b6e7202c9e78e8a2bf5333edca14f69354d276157ef00fba3a008739
  languageName: node
  linkType: hard

"object-assign@npm:^4.1.1":
  version: 4.1.1
  resolution: "object-assign@npm:4.1.1"
  checksum: fcc6e4ea8c7fe48abfbb552578b1c53e0d194086e2e6bbbf59e0a536381a292f39943c6e9628af05b5528aa5e3318bb30d6b2e53cadaf5b8fe9e12c4b69af23f
  languageName: node
  linkType: hard

"prop-types@npm:^15.6.2":
  version: 15.6.2
  resolution: "prop-types@npm:15.6.2"
  dependencies:
    loose-envify: ^1.3.1
    object-assign: ^4.1.1
  checksum: 79e478b2684449295bc8c60af1cfba4f3d414a5d832e9d23f720bce7f07df9dc52105a626134d9a3f84bb5551c9da5ec6d77d10f906b5cef26fe464959b96646
  languageName: node
  linkType: hard

"react-aria-modal@npm:^4.0.1":
  version: 4.0.1
  resolution: "react-aria-modal@npm:4.0.1"
  dependencies:
    focus-trap-react: ^8.1.0
    no-scroll: ^2.1.1
    react-displace: ^2.3.0
  peerDependencies:
    react: ^15.0.0 || ^16.0.0
  checksum: 145667b65f5df4124184eb7584b19c263b0a8d79771776a4b84395f4029600f5d2a76f07947890e889a13c0f1250da0a5690d9a2e04d71ea6a99e7f1a3a99abc
  languageName: node
  linkType: hard

"react-displace@npm:^2.3.0":
  version: 2.3.0
  resolution: "react-displace@npm:2.3.0"
  peerDependencies:
    react: 0.14.x || ^15.0.0 || ^16.0.0
    react-dom: 0.14.x || ^15.0.0 || ^16.0.0
  checksum: 45a80d1a7deb07c38dbd2180f63172598e8369f72ae8bd63cfe60f6cd221bbd84eaa43b08b8dd325b847009eb4bc109167e5367e005bbfaac120c914231863fe
  languageName: node
  linkType: hard

"react-dom@npm:^16.14.0":
  version: 16.14.0
  resolution: "react-dom@npm:16.14.0"
  dependencies:
    loose-envify: ^1.1.0
    object-assign: ^4.1.1
    prop-types: ^15.6.2
    scheduler: ^0.19.1
  peerDependencies:
    react: ^16.14.0
  checksum: 5a5c49da0f106b2655a69f96c622c347febcd10532db391c262b26aec225b235357d9da1834103457683482ab1b229af7a50f6927a6b70e53150275e31785544
  languageName: node
  linkType: hard

"react@npm:^16.14.0":
  version: 16.14.0
  resolution: "react@npm:16.14.0"
  dependencies:
    loose-envify: ^1.1.0
    object-assign: ^4.1.1
    prop-types: ^15.6.2
  checksum: 8484f3ecb13414526f2a7412190575fc134da785c02695eb92bb6028c930bfe1c238d7be2a125088fec663cc7cda0a3623373c46807cf2c281f49c34b79881ac
  languageName: node
  linkType: hard

"scheduler@npm:^0.19.1":
  version: 0.19.1
  resolution: "scheduler@npm:0.19.1"
  dependencies:
    loose-envify: ^1.1.0
    object-assign: ^4.1.1
  checksum: 73e185a59e2ff5aa3609f5b9cb97ddd376f89e1610579d29939d952411ca6eb7a24907a4ea4556569dacb931467a1a4a56d94fe809ef713aa76748642cd96a6c
  languageName: node
  linkType: hard

"tabbable@npm:^5.3.3":
  version: 5.3.3
  resolution: "tabbable@npm:5.3.3"
  checksum: 1aa56e1bb617cc10616c407f4e756f0607f3e2d30f9803664d70b85db037ca27e75918ed1c71443f3dc902e21dc9f991ce4b52d63a538c9b69b3218d3babcd70
  languageName: node
  linkType: hard

"test_project@workspace:.":
  version: 0.0.0-use.local
  resolution: "test_project@workspace:."
  dependencies:
    jquery: ^2.1.2
    react: ^16.14.0
    react-aria-modal: ^4.0.1
    react-dom: ^16.14.0
  languageName: unknown
  linkType: soft
