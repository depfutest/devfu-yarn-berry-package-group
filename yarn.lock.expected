# This file is generated by running "yarn install" inside your project.
# Manual changes might be lost - proceed with caution!

__metadata:
  version: 5
  cacheKey: 8

"focus-trap-react@npm:^3.0.4":
  version: 3.1.4
  resolution: "focus-trap-react@npm:3.1.4"
  dependencies:
    focus-trap: ^2.0.1
  peerDependencies:
    react: 0.14.x || ^15.0.0 || ^16.0.0
    react-dom: 0.14.x || ^15.0.0 || ^16.0.0
  checksum: a0b113d7b72e6f6cf971818f635398e9a7ee03fb1b08576aa17d0fb2c4f9dda234fd2413f73c59e9e13db606d18e892910ffce33596842fdb5e065b3110a051b
  languageName: node
  linkType: hard

"focus-trap@npm:^2.0.1":
  version: 2.4.6
  resolution: "focus-trap@npm:2.4.6"
  dependencies:
    tabbable: ^1.0.3
  checksum: edf31c7ddc9db611f940d1c927cc1c967f69aeb41d226d3f4f0eae2bd8a249ab4a79a523ae5bf82b958affbebbb4f1e8b0f29f7ba64db77be71bc02374929f9c
  languageName: node
  linkType: hard

"jquery@npm:^2.1.2":
  version: 2.2.4
  resolution: "jquery@npm:2.2.4"
  checksum: dc8b72b50de9b31699314e36d01fa2ed5f9b6cfa09e51c94ad07f3021e04973707247a051e317283ef96d3f10fff7542947f1916bbcbb2d9ecabff99d1a9cd58
  languageName: node
  linkType: hard

"js-tokens@npm:^3.0.0 || ^4.0.0":
  version: 4.0.0
  resolution: "js-tokens@npm:4.0.0"
  checksum: 8a95213a5a77deb6cbe94d86340e8d9ace2b93bc367790b260101d2f36a2eaf4e4e22d9fa9cf459b38af3a32fb4190e638024cf82ec95ef708680e405ea7cc78
  languageName: node
  linkType: hard

"loose-envify@npm:^1.1.0, loose-envify@npm:^1.3.1":
  version: 1.4.0
  resolution: "loose-envify@npm:1.4.0"
  dependencies:
    js-tokens: ^3.0.0 || ^4.0.0
  bin:
    loose-envify: cli.js
  checksum: 6517e24e0cad87ec9888f500c5b5947032cdfe6ef65e1c1936a0c48a524b81e65542c9c3edc91c97d5bddc806ee2a985dbc79be89215d613b1de5db6d1cfe6f4
  languageName: node
  linkType: hard

"no-scroll@npm:^2.0.0":
  version: 2.1.1
  resolution: "no-scroll@npm:2.1.1"
  checksum: fe57464f42c806ed01af7a43a56dcb9799ebef8fd3fa86d6367769b5b1952b173ef01965b6e7202c9e78e8a2bf5333edca14f69354d276157ef00fba3a008739
  languageName: node
  linkType: hard

"object-assign@npm:^4.1.1":
  version: 4.1.1
  resolution: "object-assign@npm:4.1.1"
  checksum: fcc6e4ea8c7fe48abfbb552578b1c53e0d194086e2e6bbbf59e0a536381a292f39943c6e9628af05b5528aa5e3318bb30d6b2e53cadaf5b8fe9e12c4b69af23f
  languageName: node
  linkType: hard

"prop-types@npm:^15.6.2":
  version: 15.6.2
  resolution: "prop-types@npm:15.6.2"
  dependencies:
    loose-envify: ^1.3.1
    object-assign: ^4.1.1
  checksum: 79e478b2684449295bc8c60af1cfba4f3d414a5d832e9d23f720bce7f07df9dc52105a626134d9a3f84bb5551c9da5ec6d77d10f906b5cef26fe464959b96646
  languageName: node
  linkType: hard

"react-aria-modal@npm:^2.12.1":
  version: 2.12.3
  resolution: "react-aria-modal@npm:2.12.3"
  dependencies:
    focus-trap-react: ^3.0.4
    no-scroll: ^2.0.0
    react-displace: ^2.3.0
  peerDependencies:
    react: 0.14.x || ^15.0.0 || ^16.0.0
  checksum: 5c8c6cba386a64fa41c076c2f90d91faf464b854288712fd1476df8d4eda493f39b63db5de5b2eb63813febb7eb31249d9ccf2ad484a94a36566830373faee3b
  languageName: node
  linkType: hard

"react-displace@npm:^2.3.0":
  version: 2.3.0
  resolution: "react-displace@npm:2.3.0"
  peerDependencies:
    react: 0.14.x || ^15.0.0 || ^16.0.0
    react-dom: 0.14.x || ^15.0.0 || ^16.0.0
  checksum: 45a80d1a7deb07c38dbd2180f63172598e8369f72ae8bd63cfe60f6cd221bbd84eaa43b08b8dd325b847009eb4bc109167e5367e005bbfaac120c914231863fe
  languageName: node
  linkType: hard

"react-dom@npm:^16.5.2":
  version: 16.5.2
  resolution: "react-dom@npm:16.5.2"
  dependencies:
    loose-envify: ^1.1.0
    object-assign: ^4.1.1
    prop-types: ^15.6.2
    schedule: ^0.5.0
  peerDependencies:
    react: ^16.0.0
  checksum: a53c479e8d139b3f5857f813410233b9feba300093f68ea9fa2057e91fa636bc7f97802d93cd6ae7788bad9d0114c60822e93a1fde13fb1ac396c4a83ae59127
  languageName: node
  linkType: hard

"react@npm:^16.5.2":
  version: 16.5.2
  resolution: "react@npm:16.5.2"
  dependencies:
    loose-envify: ^1.1.0
    object-assign: ^4.1.1
    prop-types: ^15.6.2
    schedule: ^0.5.0
  checksum: ed555e24327c8eefc67e9155ef4b16900300e8d12f4b6a84b0e53dc9ea4f7fbb6801a4d44c03a23cb698177e5fbb8110b12268eb95c4b32cfb906b002218aca4
  languageName: node
  linkType: hard

"schedule@npm:^0.5.0":
  version: 0.5.0
  resolution: "schedule@npm:0.5.0"
  dependencies:
    object-assign: ^4.1.1
  checksum: ab2e6329e85587a5cca36f2a6263ac40f1fb852e7ec6a43cf423c85fae29d7099913e29deb26ff2406cc807ea1dbbf7f2e015e006d02590aa0b153dec761f367
  languageName: node
  linkType: hard

"tabbable@npm:^1.0.3":
  version: 1.1.3
  resolution: "tabbable@npm:1.1.3"
  checksum: 5937c1ef89dcbf9a255b9cc0fe3493e5357fff5bd7c842260f4cbea48d32f166f4bc8a9a2f9705f9786c3d75904798cbe9bbf0e028b01241cf0eb7b635e2f3c2
  languageName: node
  linkType: hard

"test_project@workspace:.":
  version: 0.0.0-use.local
  resolution: "test_project@workspace:."
  dependencies:
    jquery: ^2.1.2
    react: ^16.5.2
    react-aria-modal: ^2.12.1
    react-dom: ^16.5.2
  languageName: unknown
  linkType: soft
